testing/unit_tests/test_patch_filter.py

```diff
@@ -0,0 +1,91 @@
+import os
+import pytest
+import numpy as np
+import torch.backends.cudnn as cudnn
+from torch.utils.data import DataLoader
+
+from ivadomed.loader.bids_dataframe import BidsDataframe
+from ivadomed import utils as imed_utils
+from ivadomed.loader import utils as imed_loader_utils, loader as imed_loader
+from testing.unit_tests.t_utils import create_tmp_dir,  __data_testing_dir__, __tmp_dir__, download_data_testing_test_files, path_repo_root
+from testing.common_testing_util import remove_tmp_dir
+
+cudnn.benchmark = True
+
+GPU_ID = 0
+BATCH_SIZE = 1
+
+
+def setup_function():
+    create_tmp_dir()
+
+
+def _cmpt_slice(ds_loader):
+    cmpt_label = {0: 0, 1: 0}
+    for i, batch in enumerate(ds_loader):
+        for gt in batch['gt']:
+            # TODO: multi label
+            if np.any(gt.numpy()):
+                cmpt_label[1] += 1
+            else:
+                cmpt_label[0] += 1
+    print(cmpt_label)
```

Comment:
I would recommend to use loguru's logger.debug/info or the like as most of the code base has migrated to those and it has auto time stamps etc (and allow centralized sink down the road if you want to keep logs etc).